ndicapimodule.cxx

```diff
@@ -185,7 +186,7 @@ bitfield_invert(PyIntObject* v)
 static PyObject*
 bitfield_lshift(PyIntObject* v, PyIntObject* w)
 {
-  register unsigned long a, b;
+  register long a, b;
```

Comment:
Is this a bug fix or a compiler warning? If compiler warning, please `#ifdef` it to OSX vs other

Reply:
Compiler mega-warning when doing (b < 0) that's impossible. These variables got assigned as from ob_digit[0] from PyLong that is a signed long and not unsigned. 

I would also remove the register keyword.